### PR TITLE
Improve the tests for PeriodicTable.tsx

### LINKED ISSUE #37: Add test for PeriodicTable

### PR DESCRIPTION
**Pull request by sudokoi**
Implement the changes to the test as per the suggestions in #38 by @tigerabrodi @mpeyper @rubenmoya

Special thanks to @mpeyper for suggesting a better way of testing this.

Closes #37

### CODE REVIEW

File: src/components/periodic-table/PeriodicTable.test.tsx [hunk 1]
```diff
@@ -19,29 +19,27 @@ function elementRenderer(atomic: number): IPeriodicTableElement {
...
-  render(<PeriodicTable elementRenderer={elementRenderer} />);
-  await waitForElementToBeRemoved(
-    () => screen.queryAllByLabelText(/loading/i),
-    { timeout: 4000 }
-  );
-}
-
 describe("should render the periodic table", () => {
   beforeAll(() => {
     ElementManager.loadElements();
```
Comment (JacobMGEvans) [inline src/components/periodic-table/PeriodicTable.test.tsx:24]: Is there a reason we need this, is the component itself not calling that getter?

**Review by JacobMGEvans — APPROVED** [review]

File: src/components/periodic-table/PeriodicTable.test.tsx (hunk 1 above)
Comment (sudokoi) [reply src/components/periodic-table/PeriodicTable.test.tsx:24]: The timeout? @JacobMGEvans

Comment (JacobMGEvans) [reply src/components/periodic-table/PeriodicTable.test.tsx:24]: ` ElementManager.loadElements();`

Comment (sudokoi) [reply src/components/periodic-table/PeriodicTable.test.tsx:24]: Yes. The PeriodicTable is being rendered inside PeriodicTablePage, which is responsible for loading the data and providing them to the component.
So, I just called the getter to sort of mock the behaviour.

**Review by emma-r-slight — COMMENTED** [review]
This looks great, especially adding the `forEach` function checking each element by name!

File: src/components/periodic-table/PeriodicTable.test.tsx (hunk 1 above)
Comment (mpeyper) [reply src/components/periodic-table/PeriodicTable.test.tsx:24]: We've mentioned in other threads that perhaps this load should be done in `setupTests` as quite a few components depend on it.

**Pull request merged**